src/views/Game.vue

```diff
@@ -100,6 +106,19 @@ export default {
     result: function() {
       return this.game && this.game.result;
     },
+    shouldRevealMrX: function() {
```

Comment:
This is basically being used to NOT reveal mr X right? method name could be misleading, that if I have understood the logic correctly

Reply:
I think the intention was for this to return a Boolean (which it does using the `some` Array helper) [which will basically return true / false]

so the `shouldRevealMrX` will be `false` for all cases except when the moveNumber is one of the revealMoves.

but if I can name this anything better to be more reflective of it's intent, let me know.